app/styles/components/_listing-page.scss

```diff
@@ -0,0 +1,351 @@
+	&--rating {
+		margin-top: calc-em(10px);
+	}
+	&--hours {
+		margin-top: calc-em(10px);
+		color: #888888;
+	}
+	&--phone {
+		margin-top: calc-em(5px);
+		color: #888888;
+	}
+	&--description {
+		margin-top: calc-em(20px);
+		border-top: 1px solid #dddddd;
+		padding-top: calc-em(20px);
+		header {
+			@extend %font-size-large;
+			font-weight: 600;
+			color: #888888;
+		}
+		p {
+			white-space: pre-line;
+			padding-right: 6px;
+			margin-top: calc-em(10px);
+		}
+	}
+}
+
+//////////////////////////////////////////////////////////////////////////////////////////
+// ORG SERVICES
+
+.service--section {
+  margin-top: calc-em(50px);
+  &--header {
+  	border-bottom: 1px solid #DDDDDD;
+  	h4 {
+  		@extend %font-size-medium;
+  		display: inline-block;
+			color: #1C72E6;
+			font-weight: bold;
+			padding-bottom: calc-em(10px);
+			border-bottom: 3px solid #1C72E6;
+		}
+  }
+
+  .service {
+	margin-top: calc-em(30px);
+	padding-top: calc-em(30px);
+	border-top: 1px solid #DDDDDD;
+	&:first-of-type {
+		border-top: 0;
+		margin-top: 0;
+	}
+	&--meta {
+		display: flex;
+		justify-content: space-between;
+		margin-right: calc-em(60px);
+		p {
+			@extend %font-size-small;
+			color: #888888;
+		}
+	}
+	&--header {
+		@extend %font-size-large;
+		font-weight: 600;
+		color: #333333;
+	}
+	&--description {
+		margin-top: calc-em(15px);
+	}
+	&--details-toggle {
+		margin-top: calc-em(20px);
+		color: #1C72E6;
+		cursor: pointer;
+		i {
+			transform: translateY(2px);
+		}
+	}
+	&--details {
+		margin-top: calc-em(40px);
+		&--item {
+			display: flex;
+			margin-top: calc-em(20px);
+			&:last-of-type {
+				.service--details--item--info {
+					border-bottom: 0;
+				}
+			}
+			header {
+				width: calc-em(130px);
+				margin-right: calc-em(20px);
+				color: #333333;
+				font-weight: 600;
+				flex-grow: 0;
+				flex-shrink: 0;
+			}
+			&--info {
+				flex-grow: 1;
+				border-bottom: 1px solid #DDDDDD;
+				padding: 0 calc-em(60px) calc-em(20px) 0;
+			}
+		}
+	}
+}
+}
+
+//////////////////////////////////////////////////////////////////////////////////////////
+// ORG INFO
+
+.info--section {
+	margin-top: calc-em(50px);
+	.info {
+		margin-top: calc-em(30px);
+		display: flex;
+		width: calc-em(600px);
+		flew-wrap: wrap;
+		&--column {
+			width: 50%;
+			@include r_max($break-tablet-s) {
+	      width: 100%;
+	    }
+	    > span {
+	    	display: block;
+	    	margin-top: calc-em(10px);
+	    	&:first-of-type {
+	    		margin-top: 0;
+	    	}
+	    }
+		}
+	}
+}
+
+.weekly-hours-list {
+	&--item {
+		display: flex;
+		line-height: 1.5;
+		&--day {
+			width: calc-em(130px);
+		}
+	}
+}
+
+.relativeTime {
+	padding: $padding-default;
+	font-weight: bold;
+	// &.warning {
+	// 	color: $color-orange;
+	// }
+
+	// &.success {
+	// 	color: $color-green;
+	// }
+
+	// &.danger {
+	// 	color: $color-red;
+	// }
+}
+
+////////////////////////////////////////////////////////////////////////////////////////
+// ORG ASIDE
+
+.listing--aside {
```

Comment:
Will remove the styles for resource after all pages have been converted to .listing styles, with consistent implementation.